syntax = "proto3";
package template;




import "model/inspection/template.proto";

/*
ListTemplate请求
*/
message ListTemplateRequest {
    /*
    套件id
    */
    string pluginId = 1 ;
    /*
    页码
    */
    int32 page = 2 ;
    /*
    每页大小
    */
    int32 pageSize = 3 ;
}

/*
ListTemplate返回
*/
message ListTemplateResponse {
    /*
    页数
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 page_size = 2 ;
    /*
    总数
    */
    int32 total = 3 ;
    /*
    数据列表
    */
    repeated inspection.InspectionTemplate list = 4 ;
}

/*
ListTemplateApi返回
*/
message ListTemplateResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    template.ListTemplateResponse data = 4 ;
}

